src/HealthChecks.Publisher.CloudWatch/HealthChecks.Publisher.CloudWatch.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkVersion)</TargetFrameworks>
```

Comment:
Do you intentionally target this project to net6 TFM only? https://www.nuget.org/packages/AWSSDK.CloudWatch#supportedframeworks-body-tab supports netstandard20 as well.

Reply:
Resolved